File: csharp-netstandard/src/Cloud.Governance.Client/Model/ChoiceType.cs


using System;
using System.Collections;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Linq;
using System.IO;
using System.Runtime.Serialization;
using System.Text;
using System.Text.RegularExpressions;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using System.ComponentModel.DataAnnotations;
using OpenAPIDateConverter = Cloud.Governance.Client.Client.OpenAPIDateConverter;

namespace Cloud.Governance.Client.Model
{
    
    [JsonConverter(typeof(StringEnumConverter))]
    
    public enum ChoiceType
    {
        [EnumMember(Value = "1")]
        DropDown = 1,

        [EnumMember(Value = "2")]
        RadioButton = 2,

        [EnumMember(Value = "3")]
        CheckBoxes = 3

    }

}

File: typescript/src/models/ChoiceType.ts
/* tslint:disable */
/* eslint-disable */
/*
 * Cloud Governance Api
 *
 * Contact: support@avepoint.com
 */

/**
 * 
 * @export
 * @enum {string}
 */
export enum ChoiceType {
    DropDown = 1,
    RadioButton = 2,
    CheckBoxes = 3
}

export function ChoiceTypeFromJSON(json: any): ChoiceType {
    return ChoiceTypeFromJSONTyped(json, false);
}

export function ChoiceTypeFromJSONTyped(json: any, ignoreDiscriminator: boolean): ChoiceType {
    return json as ChoiceType;
}

export function ChoiceTypeToJSON(value?: ChoiceType | null): any {
    return value as any;
}

